cmd/skuba/cluster/init.go

```diff
@@ -54,7 +56,15 @@ func NewInitCmd() *cobra.Command {
 				klog.Fatalf("init failed due to error: %s", err)
 			}
 		},
-		Args: cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("accepts 1 arg, received %d", len(args))
```

Comment:
This case is already covered without this change:

```
$ skuba cluster init --control-plane 1.1.1.1
** This is an UNTAGGED version and NOT intended for production usage. **
Error: accepts 1 arg(s), received 0
Usage:
...
```

```
$ skuba cluster init --control-plane 1.1.1.1 x y z
** This is an UNTAGGED version and NOT intended for production usage. **
Error: accepts 1 arg(s), received 3
Usage:
...
```

Reply:
yes, exact args tests the number of arguments. If we want a custom validation function, we are removing this cobra.ExactArgs. For consistency, I used the exact same text as cobra. However, without this patch, --control-plane 1.1.1.1 "" would incorrectly pass.

Can we just please move on?

Reply:
You're right, thanks for explaining :+1:

Reply:
Sorry for the last bit of my comment. That's why I shouldn't work during my time-off.